src/v/serde/parquet/writer.cc

```diff
@@ -20,6 +20,21 @@
 
 namespace serde::parquet {
 
+namespace {
+
+// The path within the schema starts at the root, so we can drop the root's
+// name.
+chunked_vector<ss::sstring> path_in_schema(const schema_element& element) {
+    chunked_vector<ss::sstring> path;
+    auto it = element.path.begin();
+    for (++it; it != element.path.end(); ++it) {
```

Comment:
nit: I think it = std::next(element.path.begin()) would probably be easy on the eyes ... I was staring at this for a bit :D 
(feel free to skip, not worth another round of CI).

Reply:
I'll update in the next PR I send out 😄